spec/index.html

```diff
@@ -3775,6 +3788,150 @@ <h2>Acknowledgements</h2>
+      In: Fensel, D., Sycara, K., and Mylopoulos, J. (eds.)
+      The Semantic Web - ISWC 2003. pp. 369–384.
+      Springer, Berlin, Heidelberg (2003).
+      <a href="https://doi.org/10.1007/978-3-540-39718-2_24">https://doi.org/10.1007/978-3-540-39718-2_24</a> [[HPL-2003-142]].
+    </li>
+    <li>
+      Sayers, C., Karp, A.:
+      Computing the digest of an RDF graph,
+      Tech. Rep. HPL-2003-235 (R. 1),
+      Hewlett Packard Laboratories (2004).
+      <a href="https://www.hpl.hp.com/techreports/2003/HPL-2003-235R1.html">https://www.hpl.hp.com/techreports/2003/HPL-2003-235R1.html</a>.
+    </li>
+    <li>
+      Carroll, J.J., Bizer, C., Hayes, P., Stickler, P.:
+      Named graphs.
+      Journal of Web Semantics, 3 (4), pp. 247-267,
+      World Wide Web Conference 2005——Semantic Web Track  (2005).
+      <a href="https://doi.org/10.1016/j.websem.2005.09.001">https://doi.org/10.1016/j.websem.2005.09.001</a>.
+    </li>
+    <li>
+      Tummarello, G., Morbidoni, C., Bachmann-Gmür, R., Erling, O.:
+      RDFSync: Efficient Remote Synchronization of RDF Models.
+      In: Aberer, K., Choi, K.-S., Noy, N., Allemang, D., Lee, K.-I., Nixon, L., Golbeck, J., Mika, P., Maynard, D., Mizoguchi, R., Schreiber, G., and Cudré-Mauroux, P. (eds.)
+      The Semantic Web. pp. 537–551.
+      Springer, Berlin, Heidelberg (2007).
+      <a href="https://doi.org/10.1007/978-3-540-76298-0_39">https://doi.org/10.1007/978-3-540-76298-0_39</a>.
+    </li>
+    <li>
+      Oren, E., Delbru, R., Catasta, M., Cyganiak, R., Stenzhorn, H., Tummarello, G.:
+      Sindice.com: a document-oriented lookup index for open linked data.
+      International Journal of Metadata, Semantics and Ontologies. 3, 37–52 (2008).
+      <a href="https://doi.org/10.1504/IJMSO.2008.021204">https://doi.org/10.1504/IJMSO.2008.021204</a>.
+    </li>
+    <li>
+      Fisteus, J.A., Fernández García, N., Sánchez Fernández, L., Delgado Kloos, C.:
+      Hashing and canonicalizing Notation 3 graphs.
+      Journal of Computer and System Sciences. 76, 663–685 (2010).
+      <a href="https://doi.org/10.1016/j.jcss.2010.01.003">https://doi.org/10.1016/j.jcss.2010.01.003</a>.
+    </li>
+    <li>
+      Gross, J.L., Yellen, J., Zhang, P.:
+      Handbook of Graph Theory, Second Edition.
+      Chapman & Hall/CRC (2013).
+    </li>
+    <li>
+      Kasten, A., Scherp, A., Schauß, P.:
+      [[[eswc2014Kasten]]].
+      In: Presutti, V., d’Amato, C., Gandon, F., d’Aquin, M., Staab, S., and Tordai, A. (eds.)
+      The Semantic Web: Trends and Challenges. pp. 146–160.
+      Springer International Publishing, Cham (2014).
+      <a href="https://doi.org/10.1007/978-3-319-07443-6_11">https://doi.org/10.1007/978-3-319-07443-6_11</a> [[eswc2014Kasten]].
+    </li>
+    <li>
+      Hogan, A.:
+      [[[Hogan-Canonical-RDF]]].
+      ACM Trans. Web. 11, 22:1-22:62 (2017).
+      <a href="https://doi.org/10.1145/3068333">https://doi.org/10.1145/3068333</a> [[Hogan-Canonical-RDF]].
```

Comment:
Also paywalled, I think.  But the specref link goes to authors pdf link.  Good to leave DOI link here?

Reply:
The DOI is more "official", so it's good to use as a primary reference. In this case, we had a bib entry, so adding it was easy.